tests/test_yaml_search_space/correct_config.yaml

```diff
@@ -1,32 +1,31 @@
-pipeline_space:
-  param_float1:
-    lower: 0.00001
-    upper: 0.1
-    log: TRUE
-    is_fidelity: off
+param_float1:
+  lower: 0.00001
+  upper: 0.1
+  log: TRUE
+  is_fidelity: off
 
-  param_int1:
-    lower: -3
-    upper: 30
-    log: false
-    is_fidelity: on
-  param_int2:
-    type: int
-    lower: 1E2
-    upper: 3e4
-    log: ON
-    is_fidelity: FALSE
+param_int1:
+  lower: -3
+  upper: 30
+  log: false
+  is_fidelity: on
+param_int2:
+  type: int
+  lower: 1E2
+  upper: 3e4
+  log: ON
+  is_fidelity: FALSE
 
-  param_float2:
-    lower: 3.3e-5
-    upper: 1.5E-1
+param_float2:
+  lower: 3.3e-5
+  upper: 1.5E-1
 
-  param_cat:
-    choices: [2, "sgd", 10e-3]
+param_cat:
+  choices: [2, "sgd", 10e-3]
 
-  param_const1:
-    value: 0.5
+param_const1:
+  value: 0.5
 
-  param_const2:
-    value: 1e3
-    is_fidelity: TRUE
+param_const2:
+  value: 1e3
+  is_fidelity: TRUE
```

Comment:
This will be incorrect soon fyi

Reply:
Since the idea was to pass the value directly to the key, this fits me perfectly